.github/workflows/test-external.yml

```diff
@@ -40,7 +40,8 @@ jobs:
         uses: actions/checkout@v3
         with:
           path: halmos
-          submodules: recursive
+          # we won't be needing tests/lib for this workflow
+          submodules: false
```

Comment:
@daejunpark let's keep this change though right, I think it's good and inoffensive for external tests